omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/DatabaseChangeController2_0Test.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs2_0;
+
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.openmrs.util.DatabaseUpdater;
+
+public class DatabaseChangeController2_0Test extends MainResourceControllerTest {
+
+	@Override
+	public String getURI() {
+		return "databasechange";
+	}
+
+	@Override
+	public String getUuid() {
+		try {
+			return DatabaseUpdater.getDatabaseChanges().get(0).getId();
+		}
+		catch (Exception e) {
+			return null;
+		}
+	}
+
+	@Override
+	public long getAllCount() {
+		try {
+			return Math.min(DatabaseUpdater.getDatabaseChanges().size(), 50); // page is max 50
+		}
+		catch (Exception e) {
+			return 0;
```

Comment:
Same as above.

Reply:
https://github.com/openmrs/openmrs-module-webservices.rest/pull/491#discussion_r666344449

Reply:
What happens if we do not catch this exception?

Reply:
It's a checked exception so either we wrap this call in try-catch block or add it to the method signature, that this method throws. If we add that to the method signature, then we would have to add the same throws Exception to `getUuid`, which is illegal, as it doesn't throw anything in the interface.